llvm/test/Instrumentation/RealtimeSanitizer/rtsan_unsafe.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: opt < %s -passes=rtsan -S | FileCheck %s
```

Comment:
Use update_test_checks.py, unless there's some reason this is not possible here.

Reply:
Awesome, great tool. Thanks for showing me this script - I've updated this test now